README.md

```diff
@@ -64,6 +64,10 @@ liking, making it completely customizable. This is a *self-hosted bot* – meani
 to host and maintain your own instance. You can turn Red into an admin bot, music bot, trivia bot,
 new best friend or all of these together!  
 
+Red is built to be used in [Discord](https://discord.com/), a voice, video, and text chat application that allows users to 
+communicate with their friends and communities. Discord utilizes servers that contain chat rooms and 
+voice channels that can be accessed through invite links.
```

Comment:
I think that the second sentence here may be non-obvious, "servers" in the context that Discord uses it is IMO not an everyday use of the term. "spaces" is probably better. The meaning I would try to convey here is:
> Discord allows any of its users to create spaces (servers) with text and video chat rooms (channels) for their friend groups or communities.

Part of this is already said in the first sentence so it would be good to not say the same thing twice too.